src/dtos/monitor-plan-params.dto.ts

```diff
@@ -1,30 +1,24 @@
-import { propertyMetadata } from '@us-epa-camd/easey-common/constants';
+import {
+  ErrorMessages,
+  propertyMetadata,
+} from '@us-epa-camd/easey-common/constants';
 import { IsBoolean, IsNumber, IsOptional } from 'class-validator';
 import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
 import { PaginationDTO } from './pagination.dto';
+import { IsOrisCode } from '@us-epa-camd/easey-common/pipes';
+import { Transform } from 'class-transformer';
 
 export class MonitorPlanParamsDTO extends PaginationDTO {
```

Comment:
Since it's extending paginationDTO it's adding additional parameters that are not needed